frontend/apps/filemanager/filemanager.lua

```diff
@@ -944,6 +944,7 @@ function FileManager:getSortingMenuTable()
     local fm = self
     local collates = {
         strcoll = {_("filename"), _("Sort by filename")},
+        numeric = {_("numeric"), _("Sort by filename(detect numbers)")},
```

Comment:
```suggestion
        numeric = {_("numeric"), _("Sort by filename (detect numbers)")},
```

Reply:
(Also usually called "natural sorting", FWIW).

Reply:
"Sort by filename (natural sorting)" - want me to rename it like that, or keep as is?

Reply:
Yup! (c.f. https://en.wikipedia.org/wiki/Natural_sort_order)